docs/api/Dispatcher.md

```diff
@@ -207,6 +207,7 @@ Returns: `void`
 * **onHeaders** `(statusCode: number, headers: Buffer[], resume: () => void) => boolean` - Invoked when statusCode and headers have been received. May be invoked multiple times due to 1xx informational headers. Not required for `upgrade` requests.
 * **onData** `(chunk: Buffer) => boolean` - Invoked when response payload data is received. Not required for `upgrade` requests.
 * **onComplete** `(trailers: Buffer[]) => void` - Invoked when response payload and trailers have been received and the request has completed. Not required for `upgrade` requests.
+* **onBodyChunkSent** `(chunkSize: number, totalSent: number) => void` - Invoked when a body chunk is sent to the server. Not required. For a stream body this will be invoked for every chunk. For other body types, it will be invoked once after the body is sent.
```

Comment:
> For other body types, it will be invoked once after the body is sent.

So this doesn't exactly fix the issue. It will be called only once when passing a Buffer...

Reply:
Because we pass through the buffer to the inner layers without doing any splitting ourselves.

Reply:
Ah, right, it's okay :)